docs/catalog/cloud/encrypt-what-is-necessary

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: @amexboy @gopaltayal @samudurand @shainu-x
+published_date: TBD
+category: cloud
+description: Data protection through encryption is a crucial aspect of our security measures. However, the encryption process can be resource-intensive at multiple levels.
+tags:
+- cloud
+- security
+- role:cloud-engineer
+---
+# Encrypt What Is Necessary
+
+## Description
+Data protection through encryption is a crucial aspect of our security measures. However, the encryption process can be resource-intensive at multiple levels.
+Firstly, the amount of CPU required for encryption varies depending on the chosen algorithm, and more complex algorithms tend to demand higher computational power.
+Additionally, encryption can lead to increased storage requirements as it inflates the size of the data being stored because it typically contains additional metadata and padding, which is especially noticeable for smaller files.
+Furthermore, encryption is a repetitive task that needs to be performed each time data is fetched or updated. This repetitive nature can contribute to increased energy consumption, especially in high-throughput systems.
+
+## Solution
+1. Carefully consider the sensitivity of the data that the application processes to understand if encryption is needed or not.
+2. Revise policy of what is considered sensitive and understand which environment can impact this sensitivity.
+3. Choose the appropriate algorithm and the number of iterations in case of hashing based on your application's performance and security needs.
+
+## SCI Impact
+SCI = (E * I) + M per R [Software Carbon Intensity Spec](https://grnsft.org/sci)
+Only encrypting what needs to be encrypted will impact SCI as follows:
+E: Reducing the amount of data that needs encryption will reduce the amount of energy needed to encrypt/decrypt, and store the data.
+
+## Assumptions
+We assume that a significant amount of data is not sensitive and can be stored unencrypted.
+
+## Considerations
+1. Segregate data into sensitive and non-sensitive data and only encrypt the sensitive data.
+2. Separate storage for encrypted and non-encrypted data.
+3. Hard to implement post application design.
+
+## References
+- [Comparative Analysis of Energy Costs of Asymmetric vs Symmetric Encryption-Based Security Applications](https://eprints.soton.ac.uk/469889/1/Comparative_Analysis_of_Energy_Costs_of_Asymmetric_vs_Symmetric_Encryption_Based_Security_Applications.pdf)
```

Comment:
Since this is a scientific article, DOI links should be used: https://doi.org/10.1109/ACCESS.2022.3192970 . These links are then maintained by the DOI foundation.